src/app/CMakeLists.txt

```diff
@@ -130,33 +135,33 @@ if (APPLE)
     )
     list(APPEND QBT_APP_RESOURCE_SOURCE ${QT_TRANSLATIONS})
     set_source_files_properties(${QT_TRANSLATIONS} PROPERTIES MACOSX_PACKAGE_LOCATION translations)
-endif()
-
-target_sources(qBittorrent PRIVATE ${QBT_QM_FILES} ${QBT_APP_RESOURCE_SOURCE})
-
-get_target_property(QBT_EXECUTABLE_NAME qBittorrent OUTPUT_NAME)
 
-if (APPLE)
+    get_target_property(QBT_EXECUTABLE_NAME qBittorrent OUTPUT_NAME)
     set(qbt_BUNDLE_NAME ${QBT_EXECUTABLE_NAME})
 
     # substitute @EXECUTABLE@ in dist/mac/Info.plist
     set(EXECUTABLE ${qbt_BUNDLE_NAME})
     configure_file(${qBittorrent_SOURCE_DIR}/dist/mac/Info.plist ${qBittorrent_BINARY_DIR}/dist/mac/Info.plist @ONLY)
 
     set_target_properties(qBittorrent PROPERTIES
+        MACOSX_BUNDLE ON
         MACOSX_BUNDLE_BUNDLE_NAME "${qbt_BUNDLE_NAME}"
         MACOSX_BUNDLE_INFO_PLIST ${qBittorrent_BINARY_DIR}/dist/mac/Info.plist
     )
 endif()
 
+target_sources(qBittorrent PRIVATE ${QBT_APP_RESOURCE_SOURCE})
+
 # installation
 install(TARGETS qBittorrent
     RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}
     BUNDLE  DESTINATION .
     COMPONENT runtime
 )
 
-if (GUI AND APPLE)
-    find_package(Qt5Svg REQUIRED)
-    include(bundle)
+if (CMAKE_SYSTEM_NAME MATCHES "Windows")
+    install(FILES $<TARGET_PDB_FILE:qBittorrent>
```

Comment:
There is error on Windows using MinGW:
```
CMake Error:
  Error evaluating generator expression:

    $<TARGET_PDB_FILE:qBittorrent>

  TARGET_PDB_FILE is not supported by the target linker.
```

Please fix it!

Reply:
@glassez 

Hmm, actually, I haven't touched `winconf-mingw.cmake` nor tested with building with MinGW at all. I'll fix this one error with a simple `AND NOT MINGW` in that conditional, but I would appreciate suggestions on how to improve the MinGW configuration in general - currently, it's still use stuff like `LibtorrentRasterbar_CUSTOM_DEFINITIONS`. Or at least, can you please show me how to setup an environment similar to what you're using, so that I can test properly myself? I have never used MinGW before.

Reply:
>I'll fix this one error with a simple AND NOT MINGW in that conditional

Maybe replace the condition above with `if (MSVC)`?
> I have never used MinGW before.

I have to use MinGW environment on my tablet PC (when I'm away from one of my computers), because there's very little storage to keep the Visual Studio installed. I don't care much about the productivity of this environment, just that qBittorrent is compiled and run for testing.
>can you please show me how to setup an environment similar to what you're using, so that I can test properly myself? 

I just install everything needed using official Qt installer. The only thing left for me to build myself (apart from libtorrent and qBittorrent) is a Boost.System (this is not much different from the build using MSVC).

Reply:
@glassez 

> Maybe replace the condition above with `if (MSVC)`?

:+1:

> I have to use MinGW environment on my tablet PC (when I'm away from one of my computers), because there's very little storage to keep the Visual Studio installed. I don't care much about the productivity of this environment, just that qBittorrent is compiled and run for testing.

Right, I don't need more justifications, clearly you find this useful and important. I'm just stating that I have no prior experience with it.

> > can you please show me how to setup an environment similar to what you're using, so that I can test properly myself?
> 
> I just install everything needed using official Qt installer. The only thing left for me to build myself (apart from libtorrent and qBittorrent) is a Boost.System (this is not much different from the build using MSVC).

I was expecting more of a step-by-step, but I'll give it a shot anyway.

Reply:
@glassez this particular problem has been fixed, but I'm still not able to build successfully with MinGW. I'll mark this as resolved, let's continue solving the remaining problem in the comments at the bottom.